docs/HACKING.md

```diff
@@ -300,3 +300,30 @@ It will fail if you use e.g. `:latest`.
 
 At the time of this writing, the [kubelet](https://github.com/smarterclayton/origin/blob/4de957b019aee56931b1a29af148cf64865a969b/images/os/Dockerfile)
 has code to do this in CI, and work is in progress to [replicate that for cri-o](https://github.com/openshift/release/pull/4030).
+
+### Running worker nodes with customized kubelet
+
+1) Ssh to the node (use github.com/eparis/ssh-bastion)
```

Comment:
we don't encourage sshing in to nodes

Reply:
We need elevated privileges to stop and restart kubelet, that's the reason I have this step